packages/project-account-switcher/src/presenters/ContentPresenter.js

```diff
@@ -88,6 +88,7 @@ export default class ContentPresenter extends Component {
           tabIndex="0"
         >
           <span className={css(styles.switcherAccountImageWrapper)}>
+            <img className={css(styles.image)} src={image || ""} alt={label} />
```

Comment:
There should be some conditional logic to only show the image if there is an image available otherwise do not render the markup. If there is an image then do not render the `Typography` element with the placeholder.

Currently it is rendering an empty `<img>` tag when no image is supplied, and renders both placeholder and image when there is. 

See images attached.

Reply:
Also you have two commits with the same commit message `fix: project switcher not show image in dropdown` can you rebase and squash the commits.

Reply:
Thanks, I just found in current released code, it uses this simple logic (still render empty image even there's no image provided) for showing images in the label part of this component (line 123, ProjectAccountSwitcherPresenter.js), i thought it may want to display the placeholder text before the image is loaded. Thus, i just copied the similar code to the drop-downs list (contentPresenter.js) for the images.

OK, i will implement an if-else logic for the drop-downs, and let's discuss then if we should make another fix, replace the empty image logic in ProjectAccountSwitcherPresenter.js as well.

Reply:
That is a good point about leaving the placeholder for before the image loads. I'm wondering if we should update the test to include testing for if an image is included, both a gemini and jest snapshot would work, but probably at least a jest snapshot.

Reply:
I improved the logic as
`{Boolean(image) && (<img className={css(styles.image)} src={image} alt={label} />)}`
and the typography placeholder will always render as before. 

The jest snapshots have been updated, one account(project) with image, one without.
Do you think we should update the stories as well (give an image for one of the accounts, projects)? Or we can leave it for now and update the stories as well as gemini later on? Thanks.

Reply:
And do we have the design document for this component on Wiki or other site? I want to take for a reference, I found the image style is quite simple in current code, it uses 32 * 32 for selected item, uses 60 * 30 for account list, uses 32 * 32 for project list, if the image is smaller, it will not zoom-in, if image is larger, it will only display part of the image.

Reply:
If you have time I think it would be good to update the storybook and add to gemini as well. This way we don't lose track of it later.

We do not have design on this component, but I wouldn't worry about the image re-sizing as for now I think we can expect the user to use the correct size image. Since this component has been released for a while and no one has mentioned that as an issue we can leave that for now until it gets reported as an issue.

Reply:
Hi, just dug the code a little bit, it seems for jest, for most components, we are using images from placekitten.com like "http://placekitten.com/g/32/32/" as test samples, and for storybook related tests: storybook stories and gemini stories, we are using fixtures in storybook folder. Thought we should just keep the similar usage in this component? 

Also, directly import png in jest seems not work in our current configuration, thought it may need to modify some resource name mappings for jest?